### ISSUE TITLE
ImageSharp.Web.Caching.AsyncKeyLock.GetDoorman: System.NullReferenceException

### ISSUE THREAD

**Issue by dotli** [SixLabors/ImageSharp.Web#64]
### Description
System.NullReferenceException: Object reference not set to an instance of an object.

```
at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
at SixLabors.ImageSharp.Web.Caching.AsyncKeyLock.GetDoorman(String key) in ImageSharp.Web\Caching\AsyncKeyLock.cs:line 85
at SixLabors.ImageSharp.Web.Caching.AsyncKeyLock.ReaderLockAsync(String key) in ImageSharp.Web\Caching\AsyncKeyLock.cs:line 47
at SixLabors.ImageSharp.Web.Middleware.ImageSharpMiddleware.Invoke(HttpContext context) in ImageSharp.Web\Middleware\ImageSharpMiddleware.cs:line 187
```

**Comment by James-Jackson-South** [comment]
Hey @dotli can you please use the issue template.

**Comment by JimBobSquarePants** [comment]
Hey @dotli can you please use the issue template. It makes it much easier for us to triage.

**Comment by JimBobSquarePants** [comment]
Fixed by #67
